### PR TITLE
Update dump_user_quota.py to handle inactive users

### PR DESCRIPTION
**Pull request by devincowan**
PR fixes the issue that the management command fails to run when it encounters a user that is NOT ACTIVE.

### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
Test run here:
https://new-hs-ci.hydroshare.org/view/02-Deployment/job/Production-Deployment/job/prod-hsctl-command/360/console

### CODE REVIEW

File: theme/management/commands/dump_user_quota.py [hunk 1]
```diff
@@ -71,9 +72,11 @@ def handle(self, *args, **options):
             uz_bytes = None
             dz_bytes = None
             try:
-                uz_bytes, dz_bytes = get_quota_usage(user.username)
+                uz_bytes, dz_bytes = get_quota_usage(user.username, raise_on_error=False)
```
Comment (pkdash) [inline theme/management/commands/dump_user_quota.py:75]: Why make this change? With this change I assume there would be no exception `ValidationError` which we are trying to handle here.

File: theme/management/commands/dump_user_quota.py [hunk 2]
```diff
@@ -99,7 +102,13 @@ def handle(self, *args, **options):
                 uz
             ]
             if expand:
-                quota_resources = user.uaccess.owned_resources.filter(quota_holder=user)
+                try:
+                    quota_resources = user.uaccess.owned_resources.filter(quota_holder=user)
+                except (ValidationError, PermissionDenied) as e:
```
Comment (pkdash) [inline theme/management/commands/dump_user_quota.py:107]: Why would there be `ValidationError `when querying the DB?

File: theme/management/commands/dump_user_quota.py (hunk 1 above)
Comment (devincowan) [reply theme/management/commands/dump_user_quota.py:75]: thanks

File: theme/management/commands/dump_user_quota.py (hunk 2 above)
Comment (devincowan) [reply theme/management/commands/dump_user_quota.py:107]: good call

**Review by pkdash — APPROVED** [review]

**Pull request merged**